io.github.beamerpresenter.beamerpresenter.yml

```diff
@@ -0,0 +1,65 @@
+app-id: io.github.beamerpresenter.beamerpresenter
```

Comment:
The source is fetched from https://github.com/stiglers-eponym/ which doesn't really match the app-id. And `https://github.com/beamerpresenter` doesn't exist.

tl;dr the app-id doesn't meet the requirements.

Reply:
Thank you for the very quick reply and for fixing some mistakes!
Since this will probably not be solved soon, I will close the PR for now. I'm sorry for wasting your time with this.

Reply:
it should be `io.github.stiglers_eponym.BeamerPresenter`. That's what the documentation says. And can be fixed right here too.

Reply:
What kinds of problems are you thinking of?

Reply:
It's not guaranteed that I will maintain this project forever. Defining the app-id seems quite permanent to me, so I wanted to avoid connecting it to a private account. But if this is common practice for projects that don't own a (sub)domain, I will follow the advice and use `io.github.stiglers_eponym.BeamerPresenter`.